app/controllers/publish/users_check_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Publish
+  class UsersCheckController < PublishController
+    def show
+      authorize(provider)
```

Comment:
could we move the authorize to a before_action?

Reply:
Good idea! 

Done here: https://github.com/DFE-Digital/publish-teacher-training/pull/3171/commits/1915b854a174dc708eb2593c223e7e7b1fd86728